crates/re_types_builder/src/codegen/rust/serializer.rs

```diff
@@ -574,7 +574,7 @@ fn quote_arrow_field_serializer(
                                     })
                                     .transpose()
                                 })
-                                .collect::<::re_types_core::SerializationResult<Vec<_>>>()?
```

Comment:
a `;` after a `collect` that has no `let` binding? what is going on here?

Reply:
Yeah the whole "who's responsible for the trailing ';'" is a mess in general.

I'll clean that up during the switch to arrow1, likely by introducing a new rule that if your quote block does not do the assignment, then it should not include the trailing comma.